selfdrive/controls/controlsd.py

```diff
@@ -144,6 +144,7 @@ def __init__(self, sm=None, pm=None, can_sock=None, CI=None):
       self.params.remove("ExperimentalLongitudinalEnabled")
     if not self.CP.openpilotLongitudinalControl:
       self.params.remove("ExperimentalMode")
+      self.params.remove("ExperimentalModeConfirmed")
```

Comment:
why was this added?

Reply:
Because I'm using `ExperimentalModeConfirmed` to determine whether experimental mode is available (not checking OP long/experimental long car params) If you moved from OP long to stock long you would still be able to toggle experimental mode onroad, unless this is cleared

Reply:
it should do the same checks as here: https://github.com/commaai/openpilot/blob/690999a2a6138368a93e818cbcebc984c7e58b91/selfdrive/ui/qt/offroad/settings.cc#L149